web/api/maestro_api/validation_schemas.py

```diff
@@ -37,6 +37,27 @@
     "additionalProperties": False,
 }
 
+run_all_schema = {
+    "type": "object",
+    "properties": {
+        "run_status": {"type": "string", "enum": RunStatus.list()},
+        "notes": {"type": "string"},
+        "labels": {
+            "anyOf": [
+                {
+                    "type": "array",
+                    "items": {"type": "string"},
+                },
+                {"type": "string"},
+            ]
+        },
+        "limit": {"type": "string", "minLength": 1},
```

Comment:
this is still string type since everything inside querystring is a string. I think we could think more if we can have number validation at this point. 

But, I believe it's okay for now just do that validation inside controllers